.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,52 @@
+
+name: Tests
+
+# Controls when the action will run.
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ansible:
+          - stable-2.9
```

Comment:
We're running integration tests using Ansible 5.x (Ansible-core 2.12) but lint on 2.9 and 2.10? That sounds a bit odd.